test/src/materials/TypesMaterial.C

```diff
@@ -41,6 +42,11 @@ TypesMaterial::computeQpProperties()
   _std_vec_prop[_qp][0] = 9;
   _std_vec_prop[_qp][1] = 8;
 
+  _std_vec_prop_qp[_qp].resize(1);
+  _std_vec_prop_qp[_qp][0] = 1.;
+  if (_qp == 0)
+    _std_vec_prop_qp[_qp][0] = 0.;
+
```

Comment:
I'd just test this with a generic function material that uses an x,y ramp. Then this modification becomes unnecessary.